### DISCUSSION TITLE
Is "Flax" an Acronym?

### DISCUSSION THREAD

**Discussion by jdmccaffrey** [google/flax#1742] [General]
Is "Flax" an acronym? If so, what does Flax stand for? If not, does the name Flax have some sort of meaning?
(A wild guess: Because JAX stands for "just after execution", maybe Flax derives from "flexible" + JAX).

**Comment by marcvanzee** [comment]
Indeed, Flax is FLexible jAX: "Flax: A neural network library and ecosystem for JAX designed for flexibility".

**Reply by jdmccaffrey** [reply]
Thank you!

**Comment by jdmccaffrey** [comment]
Thank you Marc!

________________________________

**Comment by avital** [comment]
I guess there are multiple correct answers! I think @jheek who wrote the first version(s) of Flax intended for it to be "Functional Layers for JAX" :)

@marcvanzee -- can we find a *third* answer?! :)
